src/js/DropContainer.jsx

```diff
@@ -0,0 +1,281 @@
+/*eslint consistent-this: [2, "that"]*/
+
+import React from 'react';
+import classNames from 'classnames';
+import accepts from 'attr-accept';
+import $ from 'jquery';
+
+import css from '../stylesheets/button.css';
+import UploadButton from './UploadButton';
+import ImagePreview from './ImagePreview';
+
+export default class DropContainer extends React.Component {
+   /**
+      * Base Container
+      * @param  {Object} props
+   */
+   constructor(props) {
+      super(props);
+      this.state = {
+         imageFile: '',
+         isDragActive: false,
+         borderStyle: '',
+         visibility: 'show'
+      };
+      this.handleChange = this.handleChange.bind(this);
+      this.onDragOver = this.onDragOver.bind(this);
+      this.onDrop = this.onDrop.bind(this);
+      this.onDragLeave = this.onDragLeave.bind(this);
+      this.onDragEnter = this.onDragEnter.bind(this);
+      this.onDragStart = this.onDragStart.bind(this);
+      // this.onClick = this.onClick.bind(this);
+   }
+
+   componentDidMount() {
+      this.enterCounter = 0;
+      // Change the background color of
+      // outer section with the prop color
+      // passing in
+      let $this = $(this.myTextInput)[0],
+         color = this.props.backgroundColor;
+      $($this).css('background-color', color);
+   }
+
+   onDragStart(e) {
+      if (this.props.onDragStart) {
+         this.props.onDragStart.call(this, e);
```

Comment:
I don't recommend using call here. Most likely the parent wants to point at it's own context and this may accidentally point to t he wrong this